tools/deploy_translatable_strings.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# Script for pushing the translatable strings to qiskit-translations repo.
+
+# Non-travis variables used by this script.
+SOURCE_REPOSITORY="git@github.com:Qiskit/qiskit-machine-learning.git"
+SOURCE_DIR=`pwd`
+SOURCE_LANG='en'
+
+TARGET_REPOSITORY="git@github.com:qiskit-community/qiskit-translations.git"
+TARGET_BRANCH_PO="master"
+
+DOC_DIR_PO="docs/locale/"
+
+echo "show current dir: "
+pwd
+
+pushd docs
+
+# Extract document's translatable messages into pot files
+# https://sphinx-intl.readthedocs.io/en/master/quickstart.html
+echo "Extract document's translatable messages into pot files and generate po files"
+tox -egettext -- -D language=$SOURCE_LANG
+
+echo "Setup ssh keys"
+pwd
+set -e
+# Add qiskit-translations push key to ssh-agent
+# TODO: Update me!
+openssl enc -aes-256-cbc -d -in ../tools/github_poBranch_update_key.enc -out github_poBranch_deploy_key -K $encrypted_deploy_po_branch_key -iv $encrypted_deploy_po_branch_iv
+chmod 600 github_poBranch_deploy_key
+eval $(ssh-agent -s)
+ssh-add github_poBranch_deploy_key
+
+# Clone to the working repository for .po and pot files
+popd
+pwd
+echo "git clone for working repo"
+git clone --depth 1 $TARGET_REPOSITORY temp --single-branch --branch $TARGET_BRANCH_PO
+pushd temp
+
+git config user.name "Qiskit (Machine Learning) Autodeploy"
+git config user.email "qiskit@qiskit.org"
+
+echo "git rm -rf for the translation po files"
+git rm -rf --ignore-unmatch $DOC_DIR_PO/$SOURCE_LANG/LC_MESSAGES/*.po \
+    $DOC_DIR_PO/$SOURCE_LANG/LC_MESSAGES/api \
+    $DOC_DIR_PO/$SOURCE_LANG/LC_MESSAGES/apidoc \
+    $DOC_DIR_PO/$SOURCE_LANG/LC_MESSAGES/theme \
+    $DOC_DIR_PO/$SOURCE_LANG/LC_MESSAGES/_*
+
+# Remove api/ and apidoc/ to avoid confusion while translating
+rm -rf $SOURCE_DIR/$DOC_DIR_PO/$SOURCE_LANG/LC_MESSAGES/api/ \
+    $SOURCE_DIR/$DOC_DIR_PO/$SOURCE_LANG/LC_MESSAGES/apidoc/ \
+    $SOURCE_DIR/$DOC_DIR_PO/$SOURCE_LANG/LC_MESSAGES/stubs/ \
+    $SOURCE_DIR/$DOC_DIR_PO/$SOURCE_LANG/LC_MESSAGES/theme/
+
+# Copy the new rendered files and add them to the commit.
+echo "copy directory"
+cp -r $SOURCE_DIR/$DOC_DIR_PO/ machine-learning/docs
+cp $SOURCE_DIR/setup.py machine-learning/.
+cp $SOURCE_DIR/requirements-dev.txt machine-learning/.
+
+# git checkout translationDocs
+echo "add to po files to target dir"
+git add $DOC_DIR_PO
```

Comment:
Is this correct? Isn't `DOC_DIR_PO` here `docs/locale` and shouldn't this be `machine-learning`?

Reply:
Right! I missed this one. I'll push the fix now

Reply:
Fixed in https://github.com/Qiskit/qiskit-machine-learning/pull/130/commits/ae168a8cf56a2d53a7a8031ea4a02039eec9439e